src/components/atom/Dropdown.tsx

```diff
@@ -1,6 +1,6 @@
 import "./style.css";
 
-const Dropdown = () => {
+const Dropdown = (): JSX.Element => {
```

Comment:
This file is not well placed, It should be in `src/components/atoms/ Dropdown/Dropdown.tsx`.
Please add a test for it.